libs/utils/Logger.lua

```diff
@@ -67,13 +68,22 @@ function Logger:log(level, msg, ...)
 	local tag = config[level]
 	if not tag then return end
 
+	local d = date(self._dateTime)
 	msg = format(msg, ...)
 
-	local d = date(self._dateTime)
+	local prettyfied = msg
```

Comment:
No need to declare a new var here. Just re-use `msg`.

Reply:
I wanted to preserve the original value of `msg` because it's the value that the method returns, and so it shouldn't include extra formatting.